test/tests/get-browser-info-test.js

```diff
@@ -0,0 +1,20 @@
+var expect         = require('chai').expect;
+var browserNatives = require('../../lib/index');
+
+describe('getBrowserInfo', function () {
+    it('Should raise an error if browser is not an alias or a path', function (done) {
+        browserNatives
+            .getBrowserInfo('ffuuuuu')
+            .then(function () {
+                throw new Error('Promise rejection expected');
+            })
+            .catch(function (err) {
+                expect(err.message).eql('Unable to find the browser. ' + '' +
```

Comment:
Strange concatenation. This string is not that long, you can keep it whole on the single line